unstructured/ingest/v2/processes/embedder.py

```diff
@@ -58,6 +58,13 @@ def get_embedder(self) -> BaseEmbeddingEncoder:
             )
 
             return VertexAIEmbeddingEncoder(config=VertexAIEmbeddingConfig(**kwargs))
+        elif self.embedding_provider == "mixedbread-ai":
+            from unstructured.embed.mixedbreadai import (
+                MixedbreadAIEmbeddingConfig,
+                MixedbreadAIEmbeddingEncoder,
+            )
+
+            return MixedbreadAIEmbeddingEncoder(config=MixedbreadAIEmbeddingConfig(**kwargs))
```

Comment:
Right now this will only map the existing options exposed in the cli: [embed.py#L17](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/v2/cli/configs/embed.py#L17), which does not include many of the options exposed by the newly introduced `MixedbreadAIEmbeddingConfig`. Do these existing options cover most of the functionality? Or should the other ones be exposed via the CI as well?

Reply:
We're going to minimize the options to match the other embedders.